CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1]
+
+### Changed
+
+- Fixed updatePackageChangelog to not to throw when there are unreleased changes ([#139](https://github.com/MetaMask/action-create-release-pr/pull/139))
```

Comment:
Although this is effectively a change in the behaviour of `updatePackageChangelog`, but it is also a fix in terms of package functionally. Thoughts on moving it to `Fixed`?